src/test/java/com/google/api/generator/gapic/composer/grpc/MockServiceImplClassComposerTest.java

```diff
@@ -30,8 +30,12 @@ public class MockServiceImplClassComposerTest {
   public static Collection<Object[]> data() {
     return Arrays.asList(
         new Object[][] {
-          {"MockEchoImpl", GrpcTestProtoLoader.instance().parseShowcaseEcho()},
-          {"MockDeprecatedServiceImpl", GrpcTestProtoLoader.instance().parseDeprecatedService()}
+          {"MockEchoImpl", GrpcTestProtoLoader.instance().parseShowcaseEcho(), "localhost:7469"},
```

Comment:
We don't need these tests if we can remove the `withDefaultHost` part from `MockServiceImplClassComposer`.

Reply:
Confirmed; removed